Cargo.toml

```diff
@@ -14,3 +14,4 @@ build = "build.rs"
 
 [dependencies]
 libc = "0.2"
+mach = "0.0"
```

Comment:
`mach_port_t` was added in version 0.0.4, so we should restrict the mach version to prevent using versions older than that. 0.0.5 fixes private in public warnings, so I'd go with that version of mach.